modules/consensus/subscribe.go

```diff
@@ -169,12 +172,17 @@ func (cs *ConsensusSet) managedInitializeSubscribe(subscriber modules.ConsensusS
 		cs.mu.RLock()
 		err = cs.db.View(func(tx *bolt.Tx) error {
 			for i := 0; i < 100 && exists; i++ {
-				cc, err := cs.computeConsensusChange(tx, entry)
-				if err != nil {
-					return err
+				select {
+				case <-cancel:
+					return errors.New("aborting managedInitializeSubscribe")
```

Comment:
Done, but for some reason it won't show up here although I pushed it to the [branch](https://github.com/NebulousLabs/Sia/compare/master...ChrisSchinnerl:CSSubscribeInterrupt?expand=1)